manifests/pagerduty.pp

```diff
@@ -21,7 +21,8 @@
     type    => 'pipe',
     source  => 'puppet:///modules/sensu_handlers/pagerduty.rb',
     config  => {
-      teams => $teams,
+      teams       => $teams,
+      environment => $environment,
```

Comment:
If you look, I had exactly the same doubts, having region environment datacenter in the params. I also had concerns of using environment for the same reason. If you see my original commit and original comment, I did raise that and had used env_region just to be safe. I spend more than half day just to use a reasonable word to replace environment.